### DISCUSSION TITLE
Pimcore 11.1.4 -  messenger_messages table is filled with pimcore_scheduled_tasks

### DISCUSSION THREAD

**Discussion by EliShklovsky** [pimcore/pimcore#16430] [General]
Hi
I am using the latest Pimcore 11.1.4 and I see that the messenger_messages table is filled with pimcore_scheduled_tasks

Although the supervisor run the following tasks:

`/bin/console messenger:consume pimcore_core pimcore_maintenance pimcore_image_optimize pimcore_search_backend_message --time-limit=300`

On Pimcore 10.x there is no need to add  messenger:consume  pimcore_scheduled_tasks

Should we add to messenger:consume the parameter:  pimcore_scheduled_tasks ?

On the Pimcore latest documentation it is written:

> The pimcore:maintenance command will add the maintenance messages to the bus and run them afterward immediately from the queue.

Best Regards
Eli

**Comment by fashxp** [comment]
Hi,
`pimcore_scheduled_tasks` definitely needs to be processed. Seems, that it is missing [here](https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Webserver_Installation#5-maintenance-cron-job), in our docker configuration, it is added, [see here](https://github.com/pimcore/skeleton/blob/11.x/.docker/supervisord.conf).
